src/constants/notifications.ts

```diff
@@ -207,6 +207,7 @@ export enum TransferNotificationTypes {
   Deposit = 'deposit',
 }
 
+// TODO: fix typo
```

Comment:
later?

Reply:
yeah i just wanted to limit the # of files this PR touches, since this touches a few other ones unrelated to this PR